CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ Please also have a look at our
 
 ### Fixed
 
+- Improve some type annotations (#1424)
```

Comment:
I don't think this should have a changelog entry, as the changes are only to an internal class and a private method.